specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2018-09-01/containerregistry_build.json

```diff
@@ -0,0 +1,1807 @@
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Registries_ScheduleRun": {
+            "$ref": "./examples/RegistriesScheduleRun.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}/getBuildSourceUploadUrl": {
+      "post": {
+        "tags": [
+          "Registries"
+        ],
+        "description": "Get the upload location for the user to be able to upload the source.",
+        "operationId": "Registries_GetBuildSourceUploadUrl",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/SourceUploadDefinition"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If the registry/build doesn't exist, 404 (Not found) is returned."
+          }
+        },
+        "x-ms-examples": {
+          "Registries_GetBuildSourceUploadUrl": {
+            "$ref": "./examples/RegistriesGetBuildSourceUploadUrl.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}/runs": {
+      "get": {
+        "tags": [
+          "Runs"
+        ],
+        "description": "Gets all the runs for a registry.",
+        "operationId": "Runs_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "description": "The runs filter to apply on the operation.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "$top",
+            "in": "query",
+            "description": "$top is supported for get list of runs, which limits the maximum number of runs to return.",
+            "required": false,
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "name": "$skipToken",
```

Comment:
How would the user get the skip token value? If the only way to get it is from a nextLink in the response, then there is limited value in documenting the parameter, since the only way that the user can get it is by parsing the URL, and then putting it back into the API to build up exactly the same URL as they got in the nextLink to begin with....

Reply:
The nextLink is self contained and can be invoked as is. Please correct me if my understanding is wrong?

Reply:
Is there some other way that I, as a user, can determine what value I could possibly pass in for the $skipToken parameter?

Reply:
No, the only way is from nextLink

Reply:
Got it